lib/jobs/ansible-job.js

```diff
@@ -78,10 +95,23 @@ function ansibleJobFactory(
      * @memberOf AnsibleJob
      */
      AnsibleJob.prototype._cleanup = function cleanup() {
+        var self = this;
+        try {
+            var data = fs.readFileSync(self.outFileName, 'utf8');
+            self.context.ansibleResultFile.push(data);
+            fs.unlink(self.outFileName, function(err) {
+                if (err) {
+                    logger.warning("Uable to remove temporary Ansible " +
+                        "result file " + self.outFileName + " " + err);
+                }
+            });
+        } catch (e) {
+            logger.info("Error reading Ansible result file " +
```

Comment:
Warning should be better that error, but if that's not a error case, we can keep using the `info` but with a little change for the message, like `no ansible result file is present` or something else.